src/App.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
-import Map from './components/Map';
+import WikiResults from './components/WikiResults';
```

Comment:
I believe I saw the [WikiResults ](https://github.com/the-collab-lab/tcl-30-whats-near-me/blob/6ca7eb1d3d012109d9783cb06e588513aa3fea92/src/components/WikiResults.js) component before 😄. I'm not sure, but I guess you will merge it after.